pkg/skaffold/deploy/status_check.go

```diff
@@ -41,9 +45,21 @@ var (
 
 	// For testing
 	executeRolloutStatus = getRollOutStatus
+
+	TabHeader = " -"
 )
 
-func StatusCheck(ctx context.Context, defaultLabeller *DefaultLabeller, runCtx *runcontext.RunContext) error {
+type Checker struct {
+	context       context.Context
+	runCtx        *runcontext.RunContext
+	out           io.Writer
+	labeller      *DefaultLabeller
```

Comment:
similarly labeller is not used anywhere (just yet, I guess, but still, it doesn't strictly belong in this PR)

Reply:
done. yeah it was copy paste from #2750